drivers/SmartThings/sonos/src/init.lua

```diff
@@ -178,12 +180,22 @@ local function scan_for_ssdp_updates(driver)
   end)
 end
 
+local startup_state_received = false
```

Comment:
We want all device activity to be more or less frozen until we get the start up state/initial contents of the augmented data store, so we have to queue the devices up.